### PR TITLE
Wrong login info warning

### PR DESCRIPTION
**Pull request by gabedealmeida**

### CODE REVIEW

**Comment by gabedealmeida** [comment]
I will move the `try catch` block into Vuex, and I'll add some logic to check if the error is an instance of SatelliteError before displaying it.

File: ui/src/store/account.js [hunk 1]
```diff
@@ -67,10 +71,26 @@ export default {
...
+					email,
+					password
+				});
+
+				token = response.token;
+			} catch (e) {
+				if (e instanceof SatelliteError) {
+					commit("setErrorMessage", { message: e.message });
+				} else {
+					console.log(e);
```
Comment (montyanderson) [inline ui/src/store/account.js:89]: Can we remove the else condition, then have `throw e` instead of return?

File: ui/src/store/account.js [hunk 2]
```diff
@@ -22,7 +23,8 @@ export default {
 			email: "",
 			token: null,
 			apiKey: null,
-			projectId: null
+			projectId: null,
+			errorMessage: ""
```
Comment (montyanderson) [inline ui/src/store/account.js:27]: We should make it null initially instead of an empty string

File: ui/src/store/account.js (hunk 1 above)
Comment (gabedealmeida) [reply ui/src/store/account.js:89]: @montyanderson Are you saying something like this?
```
} catch (e) {
  if (e instanceof SatelliteError) {
      commit("setErrorMessage", { message: e.message });
  }

  console.log(e);
  throw e;
}
```

If we're throwing this error here, are you thinking that we'll catch it in the `login` component?

Comment (montyanderson) [reply ui/src/store/account.js:89]: No log, just throw

**Review by montyanderson — APPROVED** [review]

**Pull request merged**